### DISCUSSION TITLE
Disable the Wither Storm spawn structure

### DISCUSSION THREAD

**Discussion by KWatDev** [nonamecrackers2/crackers-wither-storm-mod#2091] [Q&A] [answered]
I am making a modpack on 1.20.1 minecraft with custom content, and one of the custom items is an item that summons the Wither Storm when used, so I don't want the Wither Storm structure to spawn. I've looked through the configs and there is no mod or datapack that disables the spawn structure on curseforge, can someone tell me how to turn off the structure?

**Comment by BluePineappleGuy** [comment, marked as answer]
There is a "disable spawn house" datapack linked [here.](https://www.curseforge.com/minecraft/customization/no-rundown-house) Its not yet got a version for 1.20.1 though, you'll have to wait for it if the 1.19.4 version does not work.

**Reply by KWatDev** [reply]
Thank you!

**Reply by nonamecrackers2** [reply]
It does work for 1.20.1, I just haven't updated the file yet to include that version. I may also make it built in to the mod at some point.
